packages/filter/src/features/search-filter/stories/SearchFilter.stories.tsx

```diff
@@ -359,6 +367,14 @@ export const Demo = () => {
       </Card>
       <SearchFilterDrawer>
         <DateRangeCalendarSection
+          contentRight={
```

Comment:
Is it necessary to add this to every single story or could there be value in showing that this is an optional thing to have and display it in one story only? 🤷🏻‍♀️

Reply:
That is a possibility